### ISSUE TITLE
Figure out how the scientists want to access the quicklook products

### ISSUE THREAD

**Issue by bryan-harter** [IMAP-Science-Operations-Center/sds-data-manager#48]
For example, are PNG links on the website fine?

**Comment by tech3371** [comment]
@bourque I am ping you since students are already working on this. Can you check if you can close this?

**Comment by bourque** [comment]
Yeah we can close this. We have decided that we will be making static PNGs and hosting them on the website.
